FuncGameDev/FSharp/Behavior/CameraBehavior.fs

```diff
@@ -0,0 +1,29 @@
+﻿module CameraBehavior
+
+let updateCamera (pos : float*float) (rot : float*float) (scale : float*float) (width : float) (height : float) =  
+    let gs = GameState.instance
+
+    let cPosX = fst pos
+    let cPosY = snd pos
+
+    let cRotX = fst rot
+    let cRotY = snd rot
+
+    let cScaleX = fst scale
+    let cScaleY = snd scale
+
+
+    let cd : CameraData.T = { 
+        position = (cPosX, cPosY);
+        rotation = (cRotX, cRotY);
+        scale = (cScaleX, cScaleY);
+        width = 20.0;
+        height = 10.0 }
+    let gd : GameData.T = {
```

Comment:
This is overkill. Just use the `with` keyword. Something like 'gd = { gd with camera = cd }', I think. Should probably do that for cd, too, but since we're setting every field right now it doesn't really matter.